app/views/errors/any.html.erb

```diff
@@ -0,0 +1,4 @@
+<h3><%= @message %></h3>
+
+<p>We're sorry this <%= @code %> error occurred. If you need help, please contact OpenStax<%= " and reference error #{@error_id}" if @error_id %>.</p>
```

Comment:
Take "OpenStax" out of this -- make it configurable -- `contact <%= error_contact_name %>.  Optionally, allow to be configured with support email address so that the "contact OpenStax" can be a mailto or a link to zendesk.